client/driver/rkt.go

```diff
@@ -384,6 +419,14 @@ func (d *RktDriver) Start(ctx *ExecContext, task *structs.Task) (*StartResponse,
 	// ACI image
 	img := driverConfig.ImageName
 
+	// Check if the pod manifest passed in is valid
```

Comment:
Everywhere it seems like you are missing a `.` at the end of the sentence. I checked and it seems like they are going by that style, so I would stick to that.

Reply:
A bit confused about which style you meant, i.e.: to use dots at the end or not? In places that I checked, the project was not using dots to end sentences.

Reply:
They seem to have missed style. For documentation it always needs to be a full sentence, but people tend to stick with all comments being full sentences, up to you.